spec/requests/users_spec.rb

```diff
@@ -3,15 +3,35 @@
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/users/index'
+      get '/users/'
       expect(response).to have_http_status(:success)
     end
+
+    it 'returns render index template' do
+      get '/users/'
+      expect(response).to render_template(:index)
+    end
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if you add more tests in these controller about different scenarios in which your response could be different as well :thinking: